src/tirith/core/evaluators/contained_in.py

```diff
@@ -4,7 +4,7 @@
 from tirith.utils import sort_collections
 
 # TODO: At least add __name__ as the name for the logger
```

Comment:
Apologies, could you remove the rest `TODO` comments again, I think that commit got reset previously, thanks